File: trail1/몫 출력하기/print-share.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int cnt = 0;
        while (true) {
            int N = int.Parse(Console.ReadLine());
            if (N % 2 == 1) continue;
            Console.WriteLine(N / 2);
            cnt++;
            if (cnt == 3) break;
        }
    }
}

File: trail1/몫 출력하기/print-share.c
#include <stdio.h>

int main() {
    int cnt = 0;
    while (1) {
        int N;
        scanf("%d", &N);
        if (N % 2 == 1) continue;
        printf("%d\n", N/2);
        cnt++;
        if (cnt == 3) break;
    }
    return 0;
}